### PR TITLE
Reverse function removed from stdio.c, double printing works now.

### PR DESCRIPTION
**Pull request by angelpilled**
I removed the reverse function from stdio.c at line 68, though please correct me if I'm wrong. This shouldn't break anything.

### CODE REVIEW

**Comment by angelpilled** [comment]
The line will still be there, reverse was just commented out.

File: kernel/libc/stdio.c [hunk 1]
```diff
@@ -65,7 +65,7 @@ void ftoa(double num, char *buffer, int precision)
 	}
 
 	buffer[len + i] = '\0';
-	reverse(buffer, len + i);
+	// reverse(buffer, len + i);
```
Comment (lvntky) [inline kernel/libc/stdio.c:68]: You can delete the reverse call instead of commenting no problem at all

**Review by lvntky — CHANGES_REQUESTED** [review]

File: kernel/libc/stdio.c (hunk 1 above)
Comment (angelpilled) [reply kernel/libc/stdio.c]: Should be done now.

**Review by lvntky — APPROVED** [review]
thank you!

**Pull request merged**